src/main/java/com/hivemq/cli/mqtt/MqttClientExecutor.java

```diff
@@ -244,9 +245,11 @@ void mqtt3Publish(
     @Override
     void mqtt5Unsubscribe(final @NotNull Mqtt5Client client, final @NotNull Unsubscribe unsubscribe) {
         final String clientLogPrefix = LoggerUtils.getClientPrefix(client.getConfig());
+        final Mqtt5UserProperties userProperties = unsubscribe.getUserProperties() != null? unsubscribe.getUserProperties() : Mqtt5UserProperties.of();
```

Comment:
```suggestion
        final Mqtt5UserProperties userProperties = Objects.requireNonNullElse(unsubscribe.getUserProperties(), Mqtt5UserProperties.of());
```

Reply:
Currently the java-toolchain version is `8`, therefore no 1.9 methods can be used.
```java
// build.gradle.kts
//...
java {
    toolchain {
        languageVersion.set(JavaLanguageVersion.of(8))
    }
}
//...

Imo, this should be bumped to `11` in another ticket.